massa-models/src/config/constants.rs

```diff
@@ -211,7 +211,7 @@ pub const MAX_BOOTSTRAP_ASYNC_POOL_CHANGES: u64 = 100_000;
 /// Max bytes in final states parts
 pub const MAX_BOOTSTRAP_FINAL_STATE_PARTS_SIZE: u64 = 1_000_000_000;
 /// Max bytes in final states parts
-pub const MAX_BOOTSTRAPPED_NEW_ELEMENTS: u64 = 500;
+pub const MAX_BOOTSTRAPPED_NEW_ELEMENTS_SIZE: u64 = 100_000_000;
```

Comment:
I saw that in main you use the constant above (MAX_BOOTSTRAP_FINAL_STATE_PARTS_SIZE).
Do we still need both constants?

Reply:
One is used in `get_batch_to_stream`, the other in `get_versioning_batch_to_stream`

Reply:
```        state_new_elements_length_deserializer: U64VarIntDeserializer::new(
                Included(0),
                Included(args.max_versioning_elements_size),
```
For me there are still some incoherence there! We probably should have a state_new_elements and a versioning_new_element deserializer, with each deserializer taking its own limit, no?

Reply:
Yes indeed, it's a bit confused in the code I thought it was just the naming but no, I'll check this out

Reply:
Resolving because it's no big deal, I still see a very small inconcistancy on the config. You pass around `MAX_BOOTSTRAP_VERSIONING_ELEMENTS_SIZE` in the config, but query `MAX_BOOTSTRAP_FINAL_STATE_PARTS_SIZE` directly (unless I'm mistaken, we should pass either both or none in the config?).